catalog/conformance-tests/result-nri-cts-005.json

```
@@ -0,0 +1,14 @@
+{
+  "tested_solution": {
+    "solution_id": "nri-cts",
+    "version": "0.9.15"
+  },
+  "tested_by": {
+    "solution_name": "EcoAssist-Pro/LCA",
+    "version": "2.0.0",
+    "solution_id": "ecoassist-prolca"
+  },
+  "test_result": "passed",
+  "test_date": "2023-05-26T06:52:00Z",
+  "pathfinder_version": "1.0.0"
```

```suggestion
  "pathfinder_version": "1.0.1"
```

Thank you for creating this PR, @mill6-plat6aux. 

Your other conformance testing results are mentioning `1.0.1`. Did you perform the conformance testing against Pathfinder Tech Specs Version `1.0.0` or `1.0.1`?

---

@mill6-plat6aux - answering on behalf of @zeitgeist . Yes please submit individual PRs for independent tests you conducted with Zeroboard and boost. Thanks for your help!